gpustack/schemas/models.py

```diff
@@ -93,8 +105,23 @@ class ComputedResourceClaim(BaseModel):
     is_unified_memory: Optional[bool] = False
     offload_layers: Optional[int] = None
     total_layers: Optional[int] = None
-    memory: Optional[int] = Field(default=None)  # in bytes
-    gpu_memory: Optional[int] = Field(default=None)  # in bytes
+    ram: Optional[int] = Field(default=None)  # in bytes
+    vram: Optional[Dict[int, int]] = Field(default=None)  # in bytes
```

Comment:
Why do we change these? We still use gpu_memory in other APIs. Should we change as well?

Reply:
others will update in another PR.